cv.tex

```diff
@@ -44,92 +50,85 @@
 % Education
 %------------------------------------------------
 
-\section{Education} 
-
-\subsection{Cornell University}
-
-\descript{MEng in Computer Science}
-\location{Expected Dec 2014 | Ithaca, NY \\ Cum. GPA: N/A}
+\section{Education}
+\subsection{\href{https://www.ucf.edu/}{UCF}}
+\descript{\href{https://www.ucf.edu/degree/information-technology-bs/}{B.S. in Information Technology}}
+\location{December 2014 | Orlando, Fl}
+\href{https://www.cecs.ucf.edu/}{College of Eng. \& Computer Sciences} \\
+Dean's List (Later Semesters) \\
```

Comment:
(JC):
How important is it to you to mention “(Later Semesters)”? I think it detracts and is not
concerning to me if you’d remove it. If you made it even just once, then it qualifies to be
listed. Period.

Reply:
ty